templates/organization/organization_position_list.html

```diff
@@ -62,51 +88,87 @@ <h1>{{ organization.organization_name }}: Position List</h1>
             {{ organization.organization_twitter_handle|default_if_none:"" }}
         {% endif %} (<a href="{% url 'import_export_twitter:refresh_twitter_organization_details' organization.id %}?google_civic_election_id={{ google_civic_election_id }}">refresh Twitter details</a>)
         (<a href="{% url 'import_export_twitter:delete_images_view'%}?organization_id={{ organization.id }}&google_civic_election_id={{ google_civic_election_id }}">Delete Cached Images</a>)</td>
+
+        <th>Google Analytics account number:  </th>
+        <td>
+            {{organization.chosen_google_analytics_account_number|default_if_none:""}}
+
+        </td>
     </tr>
-    {% if organization.twitter_name %}
+
     <tr>
         <th>Twitter Descriptive Name:</th>
         <td>{{ organization.twitter_name|default_if_none:"" }}</td>
+        <th>HTML verification string: </th>
+        <td>
+            {{organization.chosen_html_verification_string|default_if_none:""}}
+        </td>
+
     </tr>
-    {% endif %}
-    {% if organization.twitter_description %}
+
     <tr>
+        {% if organization.twitter_description %}
         <th>Twitter Description:</th>
         <td>{{ organization.twitter_description|default_if_none:"" }}</td>
+        {% endif %}
+        <th> LOGO URL:</th>
+        <td>
+            {{organization.chosen_logo_url_https|default_if_none:""}}
+        </td>
     </tr>
-    {% endif %}
-    {% if organization.twitter_location %}
     <tr>
+        {% if organization.twitter_location %}
         <th>Twitter Location:</th>
         <td>{{ organization.twitter_location|default_if_none:"" }}</td>
+         {% endif %}
+
+        <th> Social share description:</th>
+        <td>
+            {{organization.chosen_social_share_description|default_if_none:""}}
+        </td>
     </tr>
-    {% endif %}
-    {% if organization.organization_facebook %}
     <tr>
         <th>Facebook: </th>
         <td>{% if organization.generate_facebook_link %}
                 <a href="{{ organization.generate_facebook_link }}" target="_blank">{{ organization.organization_facebook }}</a>
             {% else %}
             {{ organization.organization_facebook|default_if_none:"" }}
-            {% endif %}</td>
+            {% endif %}
+        </td>
+
+        <th> Social share image:</th>
+        <td>
+            {{organization.chosen_social_share_image_256x256_url_https|default_if_none:""}}
+        </td>
+
     </tr>
-    {% endif %}
-    {% if organization.organization_email %}
     <tr>
+       {% if organization.organization_email %}
```

Comment:
I would remove this `if organization.organization_email` (and the corresponding endif) so a missing organization_email doesn't shift the "Sub domain string" into the left column.